frontend/src/pages/chat/Chat.tsx

```diff
@@ -90,6 +91,15 @@ const Chat = () => {
         setActiveCitation(undefined);
         setActiveAnalysisPanelTab(undefined);
 
+        if (isFinancialAssistantActive==true){
```

Comment:
@ramirezmorac2 if the value of agent is taken here no need to keep the agent state in the `QuestionInput` component or pass it to the `makeApiRequestGpt` function